src/components/Events/EventCard.js

```diff
@@ -69,57 +73,59 @@ const AlignRightCol = styled(Col)`
 `
 
 const EventCard = ({ event }) => {
+  // TODO delete the default value for type and attendees when they can be retrieved
   const {
-    type,
     date,
-    eventName,
-    eventLocation,
+    address,
     startTime,
     endTime,
-    attendees = 0,
-    link
+    type = 'TODO - type',
+    eventTitle,
+    attendees = `TODO - ${0}`,
+    linkToEvent
   } = event
 
+  // TODO - Still need to get properly formatted address
+  const formattedAddress = address.slice(0, 15).trim()
+
+  const formattedStartTime = format(new Date(startTime), 'ha')
+  const formattedEndTime = format(new Date(endTime), 'ha')
```

Comment:
This will be an interesting exercise in dealing with timezones and date formatting!

Due to some native DOM behaviour, the time here is localised, meaning when I view the page the `require('Lx')` event is displayed as occurring between 1am - 3am. I specifically set the dates to be without timezones in contentful but it appears that contentful doesn't strip this out. I believe this require('lx') event data published by the lambda.

![image](https://user-images.githubusercontent.com/7108120/64401755-e8c5c580-d0a4-11e9-8535-83169094e7f4.png)

This is a sample of the data we get from contentful: 
```
        {
          "node": {
            "eventTitle": "K8s Connect",
            "startTime": "2019-09-11T18:00",
            "endTime": "2019-09-11T20:00",
            "date": "2019-09-11T00:00"
          }
        },
        {
          "node": {
            "eventTitle": "Require ('LX') brings  ES6 iteration protocol & Pulsarcast - a P2P Pub/Sub",
            "startTime": "2019-08-20T17:45:00.000Z",
            "endTime": "2019-08-20T19:45:00.000Z",
            "date": "2019-08-20"
          }
        },
```

Notice the require lx start and end time are appended with `.000Z` - this is ISO formatting of timezones which the DOM will take into account when formatting with the date/fns method.